x-pack/plugins/security_solution/public/timelines/components/timeline/footer/__snapshots__/index.test.tsx.snap

```diff
@@ -2,7 +2,6 @@
 
 exports[`Footer Timeline Component rendering it renders the default timeline footer 1`] = `
 <FooterContainer
-  data-test-subj="timeline-footer"
```

Comment:
this wasn't being used so i took it. Enzyme didn't seem to render this far. I expected it to (the call was to `mount`.) But I've never used enzyme so I'm probably goofing it up. This works, but if anyone has improvement suggestions let me know.